### ISSUE TITLE
Intro to Business: NOTE contents are displayed next to its callout and number sequence order continues with Chapter 17

### ISSUE THREAD

**Issue by clteam18** [Connexions/cnx-recipes#476]
**Appendix - m10302**

**Web view**
![image](https://user-images.githubusercontent.com/39794953/42884518-28fb164e-8abc-11e8-803d-869e18425899.png)

1. NOTE content displays next to its callout paragraph as shown above, and the contents could not be found in the **References** section.
2. NOTE callout number and content number continues with the sequence order of Chapter 17 and displays as (24, 25, etc.,) originally it should start with 1, 2, etc.

**Comment by openstaxalina** [comment]
@lwtchu this looks like a citation in the Appendix.  Can we support these?  I've placed this in the next milestone; please prioritize as you see fit.

Link to content: https://content02.cnx.org/contents/o0xnhd_t@4.1:anIA8Ogm@5/A--Understanding-the-Legal-and

**Comment by lwtchu** [comment]
I don't believe citations are supported in the appendix. @helenemccarron can you confirm?

**Comment by openstaxalina** [comment]
@clteam18 Since citations are not supported into the appendix, please mark these up as **footnotes** instead. This was approved by Mike Roche.  Please let us know if this approach resolves the issue.

**Comment by clteam18** [comment]
@openstaxalina Mark up has been changed to footnote element in Appendix and the screen shot below.

![image](https://user-images.githubusercontent.com/39794953/43180522-2946857c-8ff6-11e8-8ee9-6f9b69af5738.png)
